docs/conf.py

```diff
@@ -55,11 +56,14 @@
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
-#
+
+repo = git.Repo('../')
+last_commit = str(repo.head.commit)[:6]
+
 # The short X.Y version.
-version = u'1.0.0'
+version = u'1.1.0-' + last_commit
 # The full version, including alpha/beta/rc tags.
-release = u'1.0.0'
+release = u'1.1.0' + last_commit
```

Comment:
do not much care about these, curious about the  reason of making version as 0-sha and release  as 0sha